README

```diff
@@ -1 +1,2 @@
 text
+More text!
```

Comment:
Nice addition here.  Though you might want to consider this wording instead:

    Supplemental text!